src/universal/RELEASE

```diff
@@ -1,16 +1,14 @@
 Introduction
 
-This is first release candidate for the Deadalus Mantis integration release. This release integrates IOHK's Daedalus wallet
-(https://daedaluswallet.io/) with the Mantis Ethereum Classic backend.
+This is first release candidate for the Mantis 1.1 release. This release was focused on improving general performance of

```

Comment:
"This is the Mantis 1.1 release .... " We can drop the RC part, less changes when go 1.1